### PR TITLE
Move or sync profiles and verification scripts

### PR DESCRIPTION
**Pull request by rwx788**
As part of improvement of autoyast tests, we want to keep profiles and
verification scripts in single place. As same profiles are used for
integration tests, we move them to aytests repository.
Whereas, as some files were forked, we need to sync them, to be on the
same side. After merging these changes, settings of relevant jobs have
to be changed. There will be another merge required for
os-autoinst-distri-opensuse repository.

@schubi2 please review if there is any issues to run these in your setup.
ftp profile mainly has changes in formatting, and only 2 settings were changes.
For lvm, somehow openQA is not capable of other layouts than default and for example instead of ";" we print a umlaut.

Changes to openQA distri repo are suggested in [PR#3358](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3358)

### CODE REVIEW

File: aytests/bug-870820.xml [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+      <script>
+        <!-- retrieving the scripts from FTP source doesn't work since YaST seems to
+             retrieve the stuff before we have (again) network ... -->
+        <filename>02-install_scripts</filename>
+        <debug config:type="boolean">false</debug>
+        <location/>
+        <source><![CDATA[#!/bin/sh
+          mkdir /tmp/scripts
+          cd /tmp/scripts
+          wget ftp://music.arch.suse.de/scripts/after_install.sh
```
Comment (schubi2) [inline aytests/bug-870820.xml]: Hm, I fear we will not have connections to music.arch.suse in our test environment

Comment (schubi2) [reply aytests/bug-870820.xml]: Have a look to:
https://github.com/yast/aytests-tests/blob/master/aytests/tftp.vars
Perhaps we can use ENV variables instead.

File: aytests/ftp.xml [hunk 2]
```diff
@@ -1,147 +1,87 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE profile>
-<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
-  <!-- These are not supported anymore; bug 925381 -->
-  <sshd>
-    <x config:type="boolean">false</x>
```
Comment (schubi2) [inline aytests/ftp.xml:6]: Good point. That is really not needed.

File: aytests/lvm.xml [hunk 3]
```diff
@@ -140,11 +140,6 @@ mv /var/run/zypp.sav /var/run/zypp.pid
     </remove-packages>
   </software>
 
-  <language>
```
Comment (schubi2) [inline aytests/lvm.xml:143]: That is still needed for test keyboard.sh

**Review by schubi2 — CHANGES_REQUESTED** [review]
I am missing the corresponding files *.list *.rb for the new testcases bug-870820 and dns in order to get a connection between the testcases and the checking scripts.

File: aytests/bug-870820.xml (hunk 1 above)
Comment (rwx788) [reply aytests/bug-870820.xml]: Seems that part was used for some env configuration. I've tested it with openQA and it works fine: http://gershwin.arch.suse.de/tests/1233, so I've simply removed this part.

File: aytests/lvm.xml (hunk 3 above)
Comment (rwx788) [reply aytests/lvm.xml:143]: As per our discussion, I've moved this check to lvm_auto.list test, where we also set German as system language.

**Review by schubi2 — APPROVED** [review]
LGTM. Let's try it. :-)

**Pull request merged**